src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1393,10 +1393,10 @@ private async Task InternalSendRequestBodyAsync(WinHttpRequestState state, bool
             using (var requestStream = new WinHttpRequestStream(state, chunkedModeForSend))
             {
                 await state.RequestMessage.Content.CopyToAsync(
-                    requestStream,
-                    state.TransportContext
 #if HTTP_DLL
-                    , state.CancellationToken
```

Comment:
💭 It almost seems like this should be an exception, but I'm OK with the change as well.

Reply:
> It almost seems like this should be an exception, but I'm OK with the change as well.

My preference is actually that the rule takes into account ifdefs like this (can you make that happen? :wink:).  There are other rules that have this issue as well, to the extent that we won't be enabling them for now.

For this specific case, I went back and forth what to do about it, but decided the readability both ways was equally clear, and didn't want to make it more convoluted with adding pragma warning disables and restores, nor a SuppressMessage attribute.